test/stability/testDivide/testDivide.go

```diff
@@ -151,7 +151,7 @@ func executeTest(configFile, target string, testScenario uint, testType uint) er
 	}
 
 	if testScenario == receivePart {
-		inputFlow, err := flow.SetReceiver(uint8(inport1))
+		inputFlow, err := flow.SetReceiver(uint16(inport1))
```

Comment:
Maybe also convert to uint16 once, like do in other files?
inport1 = uint16(*flag.Uint(...))

Reply:
OK, yes!